src/gui/src/Action.h

```diff
@@ -36,7 +36,7 @@ class Action
     friend QTextStream& operator<<(QTextStream& outStream, const Action& action);
 
     public:
-        enum ActionType { keyDown, keyUp, keystroke, switchToScreen, switchInDirection, lockCursorToScreen, mouseDown, mouseUp, mousebutton };
+        enum ActionType { keyDown, keyUp, keystroke, switchToScreen, toggleScreen, switchInDirection, lockCursorToScreen, mouseDown, mouseUp, mousebutton };
```

Comment:
I'm wondering if the ActionType enum is converted to an integer in the protocol. Could you please verify which case it is? If we're using integral value of the enum in the protocol, then this new enum should be last.

Reply:
in src/gui/src/Action.cpp around my changes.